src/sos_polynomial_in_semialgebraic_set_bridge.jl

```diff
@@ -9,10 +9,9 @@ function lagrangian_multiplier(model::MOI.ModelLike, p, set::SOSLikeCone, q,
     # If maxdegree_s2 is odd, div(maxdegree_s2,2) would make s^2 have degree up to maxdegree_s2-1
     # for this reason, we take div(maxdegree_s2+1,2) so that s^2 have degree up to maxdegree_s2+1
     maxdegree_s = div(maxdegree_s2 + 1, 2)
-    # FIXME handle the case where `p`, `q_i`, ...  do not have the same variables
-    # so instead of `variable(p)` we would have the union of them all
-    @assert variables(q) ⊆ variables(p)
-    monos = MP.monomials(MP.variables(p), mindegree_s:maxdegree_s)
+    vars = [MP.variables(p)..., MP.variables(q)...]
+    unique!(vars)
```

Comment:
According to the docstring of `unique!`, it is more efficient to do `sort!` then `unique!`:
```julia
julia> unique!([1, 1, 1])
  1-element Array{Int64,1}:
   1
  
  julia> A = [7, 3, 2, 3, 7, 5];
  
  julia> unique!(A)
  4-element Array{Int64,1}:
   7
   3
   2
   5
  
  julia> B = [7, 6, 42, 6, 7, 42];
  
  julia> sort!(B);  # unique! is able to process sorted data much more efficiently.
  
  julia> unique!(B)
  3-element Array{Int64,1}:
    6
    7
   42
```